### ISSUE TITLE
Problem: no minimum packet timeout check for send-tx to host chain on behalf of ica account

### ISSUE THREAD

**Issue by mmsqe** [crypto-org-chain/chain-main#1070]
we need a minimum value for the relative timeout of send-tx in ica controller after migrate to ibc 8.3.x and remove icaauth module.

**Comment by yihuang** [comment]
the timeout is set by user right? what is the consequence of too low?

**Comment by mmsqe** [comment]
yes, but msg server still [ensure](https://github.com/crypto-org-chain/chain-main/blob/v4.2.9/x/icaauth/types/message_submit_tx.go#L51) this min timeout is appropriate before.

**Comment by yihuang** [comment]
> yes, but msg server still [ensure](https://github.com/crypto-org-chain/chain-main/blob/v4.2.9/x/icaauth/types/message_submit_tx.go#L51) this min timeout is appropriate before.

I mean why do we need it, what's the consequence without it.

**Comment by mmsqe** [comment]
I guess low timeout make it easier to trigger ordered channel, but I'm not sure about the history reason.

**Comment by SillyZir** [comment]
After removing `icaauth`, the `SendTx` timeout is likely user-supplied with no floor check. A `MsgSendTx` ante decorator or param-based minimum in the controller keeper would prevent zero/near-zero timeouts causing premature packet failures.
